modules/dev-tools/src/test.ts

```diff
@@ -43,6 +43,10 @@ switch (mode) {
     }
     break;
 
+  case 'node-debug':
+    runNodeTest(resolveNodeEntry('test'), '', true);
```

Comment:
boolean parameters are "evil", no way for reader to know what this means. Use a named option instead?
```suggestion
    runNodeTest(resolveNodeEntry('test'), '', {breakAndInspect: true);
```

Reply:
Done